smartseq2_single_sample/pipelines/run_pipeline_scatter.wdl

```diff
@@ -0,0 +1,53 @@
+import "hisat2_QC_pipeline.wdl" as run_hisat2
+import "hisat2_rsem_pipeline.wdl" as run_hisat2_rsem
+
+workflow run_pipelines {
+
+  # load annotation
+  File gtffile
```

Comment:
`gtffile` should have a space here (--> `gtf_file`) to be consistent with other names, as should `sampleinfo` below.